### PR TITLE
profile.d/nvm.sh: add support for nvm

### PR DESCRIPTION
**Pull request by drGrove**

### CODE REVIEW

File: profile.d/nvm.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export NVM_DIR="$HOME/.nvm"
+
+install_nvm() {
+  git clone https://github.com/creationix/nvm "$NVM_DIR"
+  cd "$NVM_DIR"
+  git checkout `git describe --abbrev=0 --tags --match "v[0-9]*" origin`
```
Comment (lrvick) [inline profile.d/nvm.sh]: nit: use $() instead of ``

Comment (lrvick) [inline profile.d/nvm.sh]: http://mywiki.wooledge.org/BashFAQ/082

**Comment by drGrove** [comment]
@lrvick this should be the last fix.

**Comment by daurnimator** [comment]
Consider cloning using `--depth` or `--single-branch`

**Comment by drGrove** [comment]
@daurnimator then you can't check out the latest tag

**Comment by drGrove** [comment]
This was  merged as 4a72becb199724434ca50f65f1c53226ce23bf22

**Pull request closed without merging**